dev/bootstrap.yaml

```diff
@@ -7,4 +7,21 @@ spec:
     - name: stolon-init
       image: apiserver:5000/stolon-bootstrap:0.0.1
       imagePullPolicy: Always
+      volumeMounts:
+        - mountPath: /var/lib/gravity/resources
+          name: resources
+        - mountPath: /etc/ssl/certs
+          name: certs
+        - mountPath: /usr/local/bin/kubectl
+          name: kubectl
+  volumes:
+  - hostPath:
+      path: /var/lib/gravity/local/packages/unpacked/gravitational.io/stolon-app/0.0.4/resources
```

Comment:
Umm, it is mounted automatically (by gravity) inside the hook container. This is for the install hook, right?